wraps/python/README.md

```diff
@@ -1,3 +1,18 @@
 # Stock Indicators for Python
 
-[Stock Indicators for Python]() is a Python wrapper for [Stock Indicators](https://github.com/DaveSkender/Stock.Indicators), written in C#.
\ No newline at end of file
+[Stock Indicators for Python]() is a Python wrapper for [Stock Indicators](https://github.com/DaveSkender/Stock.Indicators), written in C#.
+
+## Testing
+`pytest` is required for testing.
+```python
+pip install pytest
```

Comment:
What's the difference between `pip` and `pip3` and does it make a difference here?

To get it to work, after installing Python 3.8 and `pytest`, I also had to:
```
pip3 install openpyxl
```

Reply:
```suggestion
pip install pytest openpyxl
```

Do you also have Python2 installed? If so, `pip` works for Python2, `pip3`works for Python3. Usually `pip` can be replaced to `pip3`.